pkg/frontend/frontend.go

```diff
@@ -503,3 +504,37 @@ func reply(log *logrus.Entry, w http.ResponseWriter, header http.Header, b []byt
 		_, _ = w.Write([]byte{'\n'})
 	}
 }
+
+func frontendOperationResultLog(log *logrus.Entry, method string, err error) {
+	log = log.WithFields(logrus.Fields{
+		"LOGKIND":       "frontendqos",
+		"resultType":    utillog.SuccessResultType,
+		"operationType": method,
+	})
+
+	if err == nil {
+		log.Info("front end operation succeeded")
+		return
+	}
+
+	if err != nil {
+		switch err := err.(type) {
+		case *api.CloudError:
+			log = log.WithField("resultType", utillog.UserErrorResultType)
+		case statusCodeError:
+			if int(err) < 300 {
```

Comment:
I've not tested a timeout (status code 0), or whether we'd ever encounter one here, but it would be treated as success the way I read the code. Just something to consider.

Reply:
Changed it to 200 <= X < 300 so reduce possible errors that might come up as you mentioned